### PR TITLE
Add query type to DbSourceExactMatchSelector

### PR DESCRIPTION
**Pull request by wenleix**

### CODE REVIEW

**Comment by rongrong** [comment]
I feel like we should change the name of this class, but I don't have a good recommendation...

File: presto-resource-group-managers/src/main/java/com/facebook/presto/resourceGroups/db/ResourceGroupsDao.java [hunk 1]
```diff
@@ -93,8 +94,10 @@
     @SqlQuery("SELECT resource_group_id\n" +
             "FROM exact_match_source_selectors\n" +
             "WHERE environment = :environment\n" +
-            "  AND source = :source")
+            "  AND source = :source\n" +
+            "  AND (query_type IS NULL OR query_type = :query_type)")
```
Comment (nezihyigitbasi) [inline presto-resource-group-managers/src/main/java/com/facebook/presto/resourceGroups/db/ResourceGroupsDao.java]: The name of this method is slightly confusing.

The fact that this method has name `getExactMatchResourceGroup` sounds like `query_type` must also be an exact match, but the predicate `query_type IS NULL` won't make it an exact match. e.g.,  result of this method can be a resource group with a null query type even though the caller specifies a non-null one.

File: presto-resource-group-managers/src/test/java/com/facebook/presto/resourceGroups/db/TestDbSourceExactMatchSelector.java [hunk 2]
```diff
@@ -42,14 +44,36 @@ public void setup()
...
+                selector.match(new SelectionContext(true, "testuser", Optional.of("@test@test_pipeline_insert"), ImmutableSet.of("tag"), 1, Optional.empty())),
+                Optional.empty());
+        assertEquals(
+                selector.match(new SelectionContext(true, "testuser", Optional.of("@test@test_pipeline_insert"), ImmutableSet.of("tag"), 1, Optional.of(INSERT.name()))),
+                Optional.of(resourceGroupId));
+        assertEquals(
+                selector.match(new SelectionContext(true, "testuser", Optional.of("@test@test_pipeline_insert"), ImmutableSet.of("tag"), 1, Optional.of(SELECT.name()))),
+                Optional.empty());
+
+        // name not match
```
Comment (nezihyigitbasi) [inline presto-resource-group-managers/src/test/java/com/facebook/presto/resourceGroups/db/TestDbSourceExactMatchSelector.java]: nit:  // name doesn't match

**Comment by wenleix** [comment]
Per discussion with @nezihyigitbasi , we think it will be good to do exact match for now . As allowing both exact match and partial match on query type complicates things (e.g. we have same environment + source but we have both queryType = '*' and queryType = 'INSERT')

**Pull request merged**